File: PHP/pro46.php
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <?php 
        /*
            String Concatenation
            To concatenate, or combine, two strings you can use the . operator:
        */ 

              //0123456789012345678901234567890
        $ip1 = "welcome to world of php Strings";
        $ip2 = "with W3schools.com";

        $ans = $ip1.$ip2;
        echo "<hr>".$ans;

        $ans = $ip1." ".$ip2;
        echo "<hr>".$ans;

        $ans = "$ip1 $ip2";
        echo "<hr>".$ans;
    ?>
</body>
</html>

File: Python/pro46.py
ip1 = int(input("Enter value for ip1 \t"))
ip2 = int(input("Enter value for ip2 \t"))
ip3 = int(input("Enter value for ip3 \t"))

#nested

if ip1 > ip2:
    if ip1 > ip3:
        ans = ip1
    else:
        ans = ip3
else:
    if ip2 > ip3:
        ans = ip2
    else:
        ans = ip3

print("Maimum is ",ans)
